packages/daimo-common/src/foreignToken.ts

```diff
@@ -133,15 +134,27 @@ export function isNativeETH(
 
 // Any coin send (stablecoins + ETH).
 export const supportedSendCoins = new Map<string, ForeignToken>([
-  [baseUSDC.address, baseUSDC],
-  [baseDAI.address, baseDAI],
-  [baseUSDT.address, baseUSDT],
-  [baseWETH.address, baseWETH],
+  [getAddress(baseUSDC.address), baseUSDC],
+  [getAddress(baseDAI.address), baseDAI],
+  [getAddress(baseUSDT.address), baseUSDT],
+  [getAddress(baseWETH.address), baseWETH],
```

Comment:
i recommend making it so that `baseUSDC.address` etc are already correctly formatted addresses. if you want, you can use `getAddress` there